src/components/Pagination/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -1,24 +1,7 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Pagination should render correctly component 1`] = `
+exports[`Pagination should render correctly PaginationContainer 1`] = `
```

Comment:
All these changes kinda scares me 😄 

To be able to better see that nothing changed, could you split your PR in two commits, one with the conversion to ts and one with the new tests ?

Reply:
I put it at the end of the tests, like this the changes is more comprehensive.